index.html

```diff
@@ -14,27 +14,165 @@
-    </nav>
-  </header>
-  <section class="headline">
-    <h1>Hey There. I’m Lucas</h1>
-    <h2>I’m a Software Developer</h2>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-      don’t hestiate to contact me.</p>
-    <ul class="socials">
-      <li><a href="https://github.com/lucascomamala"><i class="social-icons fa-brands fa-github-alt"></i></a></li>
-      <li><a href="https://www.linkedin.com/in/lucas-comamala/"><i class="social-icons fa-brands fa-linkedin-in"></i></a></li>
-      <li><a href="#"><i class="social-icons fa-brands fa-medium"></i></a></li>
-      <li><a href="https://twitter.com/ComamalaLucas"><i class="social-icons fa-brands fa-twitter"></i></a></li>
-      <li><a href=""><i class="social-icons fa-brands fa-angellist"></i></a></li>
-    </ul>
-  </section>
+  <div id="container">
+    <header>
+      <h2 class="logo">
+        <a id="name" href="/" class="logo">Lucas Comamala</a>
+      </h2>
+      <nav>
+        <a href="#"><i class="fa fa-bars"></i></a>
+      </nav>
+    </header>
+    <section id="headline">
+      <h1 id="main-title">Hey There. I’m Lucas</h1>
+      <h2 id="main-subtitle">I’m a Software Developer</h2>
+      <p id="main-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <ul id="socials">
+        <li><a href="https://github.com/lucascomamala"><i class="social-icons fa-brands fa-github-alt"></i></a></li>
+        <li><a href="https://www.linkedin.com/in/lucas-comamala/"><i class="social-icons fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="social-icons fa-brands fa-medium"></i></a></li>
+        <li><a href="https://twitter.com/ComamalaLucas"><i class="social-icons fa-brands fa-twitter"></i></a></li>
+        <li><a href=""><i class="social-icons fa-brands fa-angellist"></i></a></li>
+      </ul>
+    </section>
+
+    <section id="works">
+      <h2 id="works-title">My Recent Works</h2>
+
+      <div id="projects">
+
+        <div id="main-project">
+          <div id="featured-img"><img src="./media/placeholder.png" alt=""></div>
+          <h3 class="mainp-title">Multi-Post Stories</h3>
+          <p class="mainp-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="mainp-tag-container">
+            <span class="mainp-tags">CSS</span>
+            <span class="mainp-tags">HTML</span>
+            <span class="mainp-tags">Bootstrap</span>
+            <span class="mainp-tags">Ruby</span>
+          </div>
+          <button class="button mainp-see" type="button">See project</button>
+        </div>
```

Comment:
- [ ] Great Job, recreating an almost identical design from the Figma template 💪 . It would be nice to orient the "see project" button and `<h3>` heading to the left instead of the center to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/176978285-0eac30b0-054b-4f37-a97c-60ef176d6253.png)  | ![image](https://user-images.githubusercontent.com/60618877/176978573-4c1b0ef6-8a4c-4bc5-b8dd-21c04d200457.png)